test/test_systems_scaffolding_scaffolder.py

```diff
@@ -81,25 +112,30 @@ def test_execute_scaffold(self, mock_prompt, mock_yaml, mock_readme, mock_dignor
             mock_single_comp, mock_list_comp
         ]
 
-        sb.systems.scaffolding.scaffolder.scaffold()
+        scaffolder = sb.systems.scaffolding.scaffolder.Scaffolder(existing=False)
+        scaffolder.scaffold()
 
         mock_prompt.assert_any_call("Enter a PROJECT NAME")
         mock_prompt.assert_any_call("Enter a PROJECT DESCRIPTION")
         mock_prompt.assert_any_call("Enter a MAINTAINER NAME")
         mock_prompt.assert_any_call("Enter a CONTACT EMAIL")
-        mock_prompt.assert_any_call("Enter a LANGUAGE", options=["Python", "R","R+Python"])
+        mock_prompt.assert_any_call("Select a LANGUAGE", options=["Python", "R", "R+Python"])
+        mock_prompt.assert_any_call("Select a TEMPLATE", options=["Default", "Dash"])
         mock_prompt.assert_any_call("Confirm Skelebot Setup", boolean=True)
 
-        mock_config.assert_called_once()
-        self.assertEqual(['foo', 'bar', 'baz'], mock_config.call_args[1]['components'])
+        mock_config.load.assert_called_once()
 
-        mock_makedirs.assert_any_call("config/", exist_ok=True)
-        mock_makedirs.assert_any_call("data/", exist_ok=True)
-        mock_makedirs.assert_any_call("models/", exist_ok=True)
-        mock_makedirs.assert_any_call("notebooks/", exist_ok=True)
-        mock_makedirs.assert_any_call("output/", exist_ok=True)
-        mock_makedirs.assert_any_call("queries/", exist_ok=True)
-        mock_makedirs.assert_any_call("src/jobs/", exist_ok=True)
+        mock_makedirs.assert_any_call("src/assets/", exist_ok=True)
+        dirname = os.path.dirname(__file__)
+        dirname = dirname[:len(dirname) - 5]
```

Comment:
This doesn't seem very safe... I think the goal is to get to skelebot's top-level directory, in which case you can just call dirname twice:
```suggestion
        dirname = os.path.dirname(os.path.dirname(__file__))
```